docs/tutorial/index.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
```

Comment:
This needs converted into a markdown file. But yeah, it didn't have any reason for being a react component page.

Reply:
Yeah I was wondering why some of the index pages are js and some are md. I'll change it!